src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -142,14 +143,25 @@ def wrapped(func):
 
 class CliCommand(object):
 
-    def __init__(self, name, handler, description=None, table_transformer=None):
+    def __init__(self, name, handler, description=None, table_transformer=None,
+                 arguments_loader=None, description_loader=None):
```

Comment:
Is this going to add further complexity for third parties seeking to onboard commands? Right now I have a vague idea what they should do. Will modules still load correctly without them (albeit more slowly)?